RealmSwift/Sync.swift

```diff
@@ -530,3 +553,490 @@ extension Realm {
         return SyncSession(for: rlmRealm)
     }
 }
+
+#if canImport(Combine)
+@available(OSX 10.15, watchOS 6.0, iOS 13.0, iOSApplicationExtension 13.0, OSXApplicationExtension 10.15, tvOS 13.0, macCatalyst 13.0, macCatalystApplicationExtension 13.0, *)
+public extension App {
+    /// :nodoc:
+    enum UserError: Error {
+        /// UserError.uncertainState - Fallback for optional errors  in Future calls. Should never happened while it's still a technically possible case.
```

Comment:
comment looks a bit wide, could prob use 2 lines, I don't think you need to specify UserError.uncertainState either